packages/graphql/src/graphql/area.js

```diff
@@ -0,0 +1,27 @@
+const { gql } = require("../util");
+
+const typeDefs = gql`
+  extend type Address {
+    districtName: String!
+    neighborhoodName: String!
+  }
+`;
+
+// Get a resolver for a specific field of accomodation
+const getResolver = (field) => async (
+  { _adressableObjectId },
+  _,
+  { loaders }
+) => (await loaders.bag.accommodation.load(_adressableObjectId))[field];
+
+const resolvers = {
+  Address: {
+    neighborhoodName: getResolver("neighborhoodName"),
+    districtName: getResolver("districtName"),
+  },
+};
```

Comment:
Let's see if we can test this part of the code.

Reply:
Yes, the is a module nog a loader. So i will do this in another PR

Reply:
why not in this PR 🤔

Reply:
Because this PR is about Matomo events. 

Testing modules is a completely new thing we are not doing yet.